src/suite_starter/suite_starter.py

```diff
@@ -35,10 +35,13 @@ class SuiteStarter:  # pylint:disable=too-many-instance-attributes
 
     announcement = None
 
-    def __init__(self):
+    def __init__(self, suite_runner_template_path: str = "/app/suite_runner_template.yaml"):
```

Comment:
Add a comment or docstring about that this path refers to the path set by the volumeMount in the deployment.

Reply:
Please verify that the message is OK

Reply:
LGTM